packages/perseus/src/util/graphie.test.ts

```diff
@@ -1,4 +1,5 @@
 // eslint-disable-next-line import/no-extraneous-dependencies
+import $ from "jquery";
```

Comment:
Yep.  It's used in our production code and I needed to mock it for tests. I think we could probably move to using `getBoundingClientRect`, but I wanted to avoid production code changes as much as possible in this PR.